CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.13) #TODO Find actual min version required
+
+find_program(PYTHON "python3")
```

Comment:
See [this page](https://cmake.org/cmake/help/v3.13/module/FindPython.html) for how to find Python using CMake

Reply:
Fixed, lowered min version to 3.12, which is when FindPython was added. However, CMake 3.12 is not available in standard Ubuntu repos before 19.04 Disco, users will have to install manually or add the PPA. Ubuntu 20.04 Focal will be the first LTS Ubuntu with the correct version in the standard repos.

Reply:
@AutonomicPerfectionist CMakePP itself requires 3.14 for all the fetch content bells and whistles so 3.12 is not a problem.